bot/seasons/valentines/whoisvalentine.py

```diff
@@ -0,0 +1,51 @@
+import json
+import logging
+from pathlib import Path
+from random import choice
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+with open(Path("bot", "resources", "valentines", "valentine_facts.json"), "r") as file:
+    FACTS = json.load(file)
+
+
+class ValentineFacts:
+    def __init__(self, bot):
+        self.bot = bot
+
+    @commands.command(aliases=('whoisvalentine', 'saint_valentine'))
+    async def who_is_valentine(self, ctx):
+        """
+        Displays info about Saint Valentine.
+        """
+        embed = discord.Embed(
+            title="Who is Saint Valentine?",
+            description=FACTS['whois'],
+            color=discord.Color.dark_magenta()
```

Comment:
We've defined [`bot.Constants.Colours.pink`](https://github.com/python-discord/seasonalbot/blob/master/bot/constants.py#L67) to use for this event.

Reply:
Done